scripts/start-https.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
```

Comment:
Suggest adding a comment here to make clear this script is used on developer workstations only so we can test locally using https.